quickwit-cli/src/split.rs

```diff
@@ -89,6 +89,18 @@ pub fn build_split_command<'a>() -> Command<'a> {
                         .required(false),
                 ])
             )
+        .subcommand(
+            Command::new("delete")
+                .about("Removes one or multiple splits from an index.")
+                .args(&[
+                    arg!(--config <CONFIG> "Quickwit config file").env("QW_CONFIG"),
+                    arg!(--index <INDEX> "ID of the target index"),
+                    arg!(--splits <SPLITS> "Comma-separated list of split IDs.")
```

Comment:
Really, this should be `<SPLIT IDs>` but clap is not happy with whitespace here.